src/pages/authentication/Login.jsx

```diff
@@ -1,51 +1,116 @@
-import "./authentication.css";
-import { Navbar } from "../../components";
+import "./Auth.css";
+import { Link, useNavigate } from "react-router-dom";
+import { TextInput, PasswordInput } from "./components";
+import { useState } from "react";
+import { useAuth } from "../../context";
+import axios from "axios";
 
 export default function Login() {
+  const [userData, setUserData] = useState({
+    email: "",
+    password: "",
+  });
+
+  const navigate = useNavigate();
+  const { authDispatch } = useAuth();
+
+  const doLoginNetworkCall = async () => {
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: userData.email,
+        password: userData.password,
+      });
```

Comment:
You could create a `loginService` service function in `/services` folder and name the file as `login.service.js` and import that here and use it for network calls. This is a good convention to follow and helps in the maintainability of code.

Maybe [this](https://medium.com/nerd-for-tech/react-js-services-854be54a6ba1) could help.